tests/Doctrine/Tests/DBAL/Functional/Schema/SchemaManagerFunctionalTestCase.php

```diff
@@ -51,10 +52,19 @@ public function testDropsDatabaseWithActiveConnections()
 
         $this->_sm->dropAndCreateDatabase('test_drop_database');
 
-        $this->assertContains('test_drop_database', $this->_sm->listDatabases());
+        $knownDatabases = $this->_sm->listDatabases();
+        if ($this->_conn->getDatabasePlatform() instanceof OraclePlatform) {
```

Comment:
well - the table name is uppercased and all tables are returned uppercase - on the api usage side I see no way to know if upper or lowercase is expected.

but maybe I'm not deep enough in this oracle fun :speak_no_evil: